src/vendors/oasisscan/swagger3.yml

```diff
@@ -504,3 +548,25 @@ components:
         - epochLeft
         - validatorName
         - icon
+    CtxRow:
+      type: object
+      properties:
+        amount:
+          type: integer
+        method:
+          type: string
+          enum:
+            - consensus.Deposit
+            - consensus.Withdraw
+            - consensus_accounts.Parameters
+            - consensus.Balance
+            - consensus.Account
```

Comment:
Nice. Is there a way we can verify this list is complete?

Reply:
(for non-paratime tx types we have: https://github.com/oasisprotocol/oasis-wallet-web/blob/25d70b387a1884daa29b523bb58ccd452ef5e3f1/cypress/integration/api-transaction-methods.ts#L34-L49 )

Reply:
I have no idea how to validate these methods. I added a comment that methods API endpoint does not include consensus methods.